Makefile

```diff
@@ -43,6 +43,10 @@ update-codegen-crds: update-scripts
 lint:
 	hack/golangci-lint.sh run --new-from-rev=${PULL_BASE_SHA}
 
+.PHONY: lint-since-master
+lint-since-master:
+	hack/golangci-lint.sh run --new-from-rev=master
+
```

Comment:
Instead of adding a `lint-since-master` target, you could set the default for the `PULL_BASE_SHA` environment variable for the `lint` target to `master`. Something like:
```make
.PHONY: lint
# Set the PULL_BASE_SHA variable to master by default to make it easier
# to use in a local development environment. Prow will override this in CI.
PULL_BASE_SHA := master
lint:
    hack/golangci-lint.sh run --new-from-rev=${PULL_BASE_SHA}
```

I can't quite remember if you want `PULL_BASE_SHA := ...` or `PULL_BASE_SHA ?= ...` in this case.

This way there are less make targets to choose from and to maintain :P

An example of where I've done this recently - https://github.com/operator-framework/operator-controller/blob/e51c0c205ec09e51691579cacbedc3ac53e4bd54/Makefile#L133-L138 and how it is overridden in CI - https://github.com/operator-framework/operator-controller/blob/e51c0c205ec09e51691579cacbedc3ac53e4bd54/.github/workflows/crd-diff.yaml#L16-L17